src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -81,23 +99,20 @@ public boolean isInWatch() {
     return isInWatch;
   }
 
-  private byte[][] joinParameters(byte[] first, byte[][] rest) {
-    byte[][] result = new byte[rest.length + 1][];
-    result[0] = first;
-    System.arraycopy(rest, 0, result, 1, rest.length);
-    return result;
-  }
-
-  private byte[][] joinParameters(byte[] first, byte[] second, byte[][] rest) {
-    byte[][] result = new byte[rest.length + 2][];
-    result[0] = first;
-    result[1] = second;
-    System.arraycopy(rest, 0, result, 2, rest.length);
-    return result;
+  /**
+   * @param user
+   * @deprecated This method will be removed in future.
+   */
+  @Deprecated
+  public void setUser(final String user) {
+    this.user = user;
   }
 
-  public void setUser(final String user) { this.user = user; }
-
+  /**
+   * @param password
+   * @deprecated This method will be removed in future.
```

Comment:
@sazzad16 please point to the alternative Constructor

Reply:
@gkorland There won't be any direct alternate.

Reply:
@gkorland There won't be any direct alternate. Other than config class and, may be, factory class, no other class will hold user data.

Reply:
So you should state it in the JDoc